contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -423,6 +423,8 @@ ON Base.rolname = Ext.rolname
 LEFT OUTER JOIN pg_catalog.pg_roles Base2
 ON Ext.login_name = Base2.rolname
 WHERE Ext.database_name = DB_NAME()
+  AND (Ext.orig_username IN ('dbo', 'db_owner', 'guest', 'sys', 'INFORMATION_SCHEMA', 'public', 'sys') -- system users should always be visible
+  OR pg_has_role(Ext.rolname, 'MEMBER')) -- Current user should be able to see users it has permission of
```

Comment:
Can we remove INFORMATION_SCHEMA, public, sys from this list? Looks like they are covered in other part of UNION.